superset/models/sql_types/presto_sql_types.py

```diff
@@ -91,3 +95,35 @@ def python_type(self) -> Optional[Type[Any]]:
     @classmethod
     def _compiler_dispatch(cls, _visitor: Visitable, **_kw: Any) -> str:
         return "ROW"
+
+
+class TimeStamp(TypeDecorator):
+    """
+    A type to extend functionality of timestamp data type.
+    """
+
+    impl = TIMESTAMP
+
+    @classmethod
+    def process_bind_param(cls, value: str, dialect: Dialect) -> str:
+        """
+        Used for in-line rendering of TIMESTAMP data type
+        as Presto does not support automatic casting.
+        """
+        return "TIMESTAMP '%s'" % value
```

Comment:
Sorry for not picking this up sooner (and surprised `pylint` didn't). This type of string formatting is dated could we use the suggestion instead?

```suggestion
        return f"TIMESTAMP '{value}'"
```